.github/workflows/ci-workflow.yml

```diff
@@ -23,6 +23,20 @@ jobs:
       - name: test
         run: CC=gcc CXX=g++ make test
 
+  gcc-make-debug:
+    name: gcc make debug
+    needs: [ clang-formatting-check ]
+    runs-on: kuzu-self-hosted-testing
+    steps:
+      - uses: actions/checkout@v2
+      - run: pip install -r tools/python_api/requirements_dev.txt
```

Comment:
For all the `pip install` on CI, we should change to `pip install --user -r`

Reply:
done